compaction_test.go

```diff
@@ -329,9 +310,7 @@ func TestCompaction(t *testing.T) {
 							"tests assume only a single granule as input",
 						)
 						cfg := table.db.columnStore.compactionConfig
-						if asArrow {
-							cfg.l1ToGranuleSizeRatio = 0.6 // use a different ratio for arrow records
-						}
+						cfg.l1ToGranuleSizeRatio = 0.6 // use a different ratio for arrow records
```

Comment:
nit: is the comment still relevant?

Reply:
Kind of? The test needs that ratio but it isn't using a different one since now there's only the one